### PR TITLE
Moved the Census blueprint to the mono repo and adjusted the cli to a…

### PR DESCRIPTION
**Pull request by johnathan-rodriguez**
…llow for an optional wait parameter

### CODE REVIEW

File: shipyard_blueprints/census/pyproject.toml [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
+#readme = "README.md"
 packages = [{include = "shipyard_census"}]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-requests = "2.28.0"
+requests = "2.31.0"
 
 [tool.poetry.group.dev.dependencies]
 shipyard-bp-utils = {path = "../shipyard-bp-utils"}
```
Comment (wrp801) [inline shipyard_blueprints/census/pyproject.toml:15]: Add `develop = true` to any path dependency

**Review by wrp801 — CHANGES_REQUESTED** [review]
Overall looks good, just a tiny change to make in the toml file

**Comment by wrp801** [comment]
@johnathan-rodriguez any reason why you deleted the check_sync_cli.py file?

**Comment by johnathan-rodriguez** [comment]
@wrp801 I thought we were removing the check sync and combining it with the trigger sync cli- do we want both for the new way?

**Comment by johnathan-rodriguez** [comment]
the old check sync would exist in the old way of doing things so the legacy function should be intact

**Comment by wrp801** [comment]
Okay so what you’re proposing is to leave the check syncs tied to the old repos?

**Comment by johnathan-rodriguez** [comment]
yes

**Review by wrp801 — APPROVED** [review]

**Pull request merged**